src/test/java/bio/terra/service/snapshot/flight/SnapshotAuthzIamStepTest.java

```diff
@@ -92,8 +100,6 @@ void testDoAndUndoStepWithDUOS() throws InterruptedException {
     StepResult doResult = step.doStep(flightContext);
     assertThat(doResult.getStepStatus(), equalTo(StepStatus.STEP_RESULT_SUCCESS));
 
-    SnapshotRequestModelPolicies policies = iamService.deriveSnapshotPolicies(snapshotRequestModel);
-    policies.addReadersItem(DUOS_FIRECLOUD_GROUP.getFirecloudGroupEmail());
```

Comment:
Not for reviewers: I'm not sure what the original intention of these two lines of code were, but they don't appear to be doing anything.